webrtc-stats-testing.md

```diff
@@ -0,0 +1,46 @@
+
+This documents describes a test suite that ensures a correct implementation. There are three methods in which we can test the API:
++ API Semantics and Verification: web-platform tests, this checks the correctness of the API semantics and availability of metrics. Most importantly, these tests checks the availability of metrics, and if the metrics are reported in the right format, i.e., in the correct units. However, in some cases, a few metrics may not be fully tested, e.g., Round Trip Time (RTT).
++ Conformance across browsers: Using KITE [KITE], which is a test engine designed to test WebRTC interoperability across browsers. This is especially important to verify if the metrics reported in the RTP Control Protocol (RTCP) Sender Report (SR), Receiver Report (RR), or Extended Reports (XR) are reported correctly. 
++ Correctness and Validation of metrics: Using KITE with a test harness for media and network, checks the correctness of the reported stats.
+
+Tests are split in three parts: API semantics, availability of metrics, and correctness of metrics.
+
+## API Semantics
+
+Tests if the getStats API returns the correct stats object depending on the API input (i.e., filter by senders, etc). Also these tests should check if the reported statistics conform to the peerconnection state machine, i.e., check when obsolete stats are removed when the conditions are met, etc. The test should be carried out in a way that all the RTCStatsType dictionaries are covered, however, in some cases it may make sense to test parts of the media pipeline as a whole, for example sender and receiver transport and media (audio and video) stats.
+
+Web Platform Tests [WPT] currently runs these tests, available at: https://wpt.fyi/results/webrtc-stats. There are also browser specific tests [WPT-CHROME].
```

Comment:
Add some words about the current test coverage of WPT (good, bad or ugly, not percentages).